### DISCUSSION TITLE
Merged a PR without review

### DISCUSSION THREAD

**Discussion by jbisits** [jbisits/QGTracerAdvection#18] [General]
Hi @navidcy,

I merged a pull request without asking for a review from you. The reason for this (and I hope that is ok) is that nothing new was added except for a folder for the code I will use in my thesis and finishing the unfinished function `MeasureMixing.jl`. With regards to this though I am not sure how useful this function is (or even if I will use it at all) so it may be that I remove this at some point.

My apologies if I should have just sent the PR through for a review. If you would like me to do so for all in the future for all PR's let me know. I will of course send anything with major changes through I just thought I would save you the time as I am not sure how useful the updates are.

Joey

**Comment by navidcy** [comment]
There is no problem Joey. Just ask review from me when you think my feedback would be useful or when you are stuck with something.

**Reply by jbisits** [reply]
Will do and thanks very much Navid!
